src/main/java/step2/model/Rank.java

```diff
@@ -0,0 +1,37 @@
+package step2.model;
+
+public enum Rank {
+	FIRST(6, 2000000000),
```

Comment:
Java 7부터는 `100_000_000`처럼 숫자에 밑줄을 사용하여 가독성을 높일 수 있어요.

```suggestion
	FIRST(6, 2_000_000_000),
```